.circleci/config.yml

```diff
@@ -1,24 +1,41 @@
 version: 2.1
 orbs:
   node: circleci/node@4.7.0
-jobs:
+
+commands:
   build-and-test:
-    machine:
-      image: ubuntu-2004:202107-02
-    environment:
-      NODE_OPTIONS: "--max_old_space_size=4096"
     steps:
       - checkout
       - node/install-npm:
-          version: "6"
+          version: '6'
       - node/install-packages
       - run: npm run lint
       - run: npm run build
       - run:
+          name: Test
           command: npm run test
           no_output_timeout: 30m
+jobs:
+  with-go-ipfs:
+    machine:
+      image: ubuntu-2004:202107-02
+    environment:
+      NODE_OPTIONS: '--max_old_space_size=4096'
```

Comment:
nit: can probably remove this option from go-ipfs

Reply:
ok, will see, if it works :)

Reply:
it does.

Reply:
yeah, `NODE_OPTIONS` is an env var used by the nodejs runtime, it's not ipfs-specific at all, so I can't imagine it having any affect at all on a go-ipfs process